graph/typeDefs.graphql

```diff
@@ -977,6 +1035,91 @@ type EditCommentResponse implements Response {
   errors: [UserError!]
 }
 
+# UpdateSettingsInput is the input used to input the global site settings. This
+# will override the existing settings, so all fields must be included.
+input UpdateSettingsInput {
+
+  # moderation is the moderation mode for all Asset's on the site.
+  moderation: MODERATION_MODE!
+
+  # Enables a requirement for email confirmation before a user can login.
+  requireEmailConfirmation: Boolean
+
+  # infoBoxEnable will enable the Info Box content visible above the question
+  # box.
+  infoBoxEnable: Boolean
+
+  # infoBoxContent is the content of the Info Box.
+  infoBoxContent: String
+
+  # questionBoxEnable will enable the Question Box's content to be visible above
+  # the comment box.
+  questionBoxEnable: Boolean
+
+  # questionBoxContent is the content of the Question Box.
+  questionBoxContent: String
+
+  # premodLinksEnable will put all comments that contain links into premod.
+  premodLinksEnable: Boolean
+
+  # questionBoxIcon is the icon for the Question Box.
+  questionBoxIcon: String
+
+  # autoCloseStream when true will auto close the stream when the `closeTimeout`
+  # amount of seconds have been reached.
+  autoCloseStream: Boolean
+
+  # customCssUrl is the URL of the custom CSS used to display on the frontend.
+  customCssUrl: String
+
+  # closeTimeout is the amount of seconds from the created_at timestamp that a
+  # given asset will be considered closed.
+  closeTimeout: Int
+
+  # closedMessage is the message shown to the user when the given Asset is
+  # closed.
+  closedMessage: String
+
+  # charCountEnable is true when the character count restriction is enabled.
+  charCountEnable: Boolean
+
+  # charCount is the maximum number of characters a comment may be.
+  charCount: Int
+
+  # organizationName is the name of the organization.
+  organizationName: String
+
+  # editCommentWindowLength is the length of time (in milliseconds) after a
+  # comment is posted that it can still be edited by the author.
+  editCommentWindowLength: Int
+}
+
+# UpdateSettingsResponse contains any errors that were rendered as a result
+# of the mutation.
+type UpdateSettingsResponse implements Response {
+
+  # An array of errors relating to the mutation that occurred.
+  errors: [UserError!]
+}
+
+# UpdateWordlistInput is the list of words that composes the Wordlist.
+input UpdateWordlistInput {
```

Comment:
This means both `banned` and `suspect` must be provided right?

Reply:
That's correct.